Fixes #3370 Evaluation request - show active evaluation request

RazgrizX opened the pull request:
Fixes #3370 
Adds a submitted icon if the workspace assignment stae is PENDING, PENDING_PASS or PENDING_FAILED. In last two cases it will be added to the grade, so all the information will be available from table of records.
Progress will not appear if workspace have been submitted.
Needs sccs changes (working on it)
An evaluation request date might be useful, but there is no endpoint to get it (The only one requires access to evaluation). A possibility to create one could be considered.

RazgrizX commented:
Do not merge if not necessary, still improving this feature

pkukkon commented:
Yup, probably best to merge only after the date of the latest state is also present in all possible cases. Postponing review/merge until @jjlankinen has had his say.

RazgrizX commented:
added date to the assessmentState. changed files where it is used. formatted it. fetched it into the submitted icon title. changed localization. Unsure about WorkspaceBackingBean.java file changes.
